.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+
+python:
+  - "2.7"
+
+before_install:
+    - pip install pep8
+
+install:
+    - python setup.py -q install
+
+script:
+    # Don't fail if a line is longer than 80 characters (E501)
+    - find . -name \*.py -exec pep8 --ignore=E501 --exclude='*/procedural_city_generation/*' {} +
```

Comment:
We should still try to enforce using the 80 char limit, but there are times when it makes more sense to have a line a bit larger if it improves legibility. We can enforce it if you think that it makes sense.